mm2src/mm2_bitcoin/spv_validation/src/conf.rs

```diff
@@ -0,0 +1,173 @@
+use crate::helpers_validation::SPVError;
+use crate::work::{DifficultyAlgorithm, RETARGETING_INTERVAL};
+use chain::{BlockHeader, BlockHeaderBits};
+use primitives::hash::H256;
+use serde::Deserialize;
+use std::num::NonZeroU64;
+use std::str::FromStr;
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct StartingBlockHeader {
+    // Starting block height
+    pub height: u64,
+    // Hash of the starting block header.
+    pub hash: Option<String>,
+    // Time of the starting block header.
+    pub time: u32,
+    // Bits of the starting block header.
+    pub bits: Option<u32>,
```

Comment:
Before using `hash`/`time`/`bits`, the header hex was non-optional. Why would we need some optional parameters and some required for the fields that replaced the header hex?

Reply:
I had something else in mind before this changes.. I'll correct!